examples/ippool/ippool.yaml

```diff
@@ -1,35 +1,40 @@
 ---
-apiVersion: cluster.x-k8s.io/v1alpha3
+apiVersion: cluster.x-k8s.io/v1alpha4
 kind: Cluster
 metadata:
   name: ${CLUSTER_NAME}
+  namespace: ${NAMESPACE}
 spec:
   clusterNetwork:
     services:
       cidrBlocks: ["10.96.0.0/12"]
     pods:
-      cidrBlocks: ["192.168.0.0/16"]
-    serviceDomain: "cluster.local"
```

Comment:
I might be missing the context, do we remove this(serviceDomain) also?

Reply:
Thanks for review @kashifest!  Yes, we do here, since I have not seen `serviceDomain` in the cluster templates nor in the master CI manifests or in PRs'. However, it does not hurt either way if we remove or keep it here, and field is not deprecated or anything, it is still supported in [CAPI](https://github.com/kubernetes-sigs/cluster-api/blob/master/test/infrastructure/docker/examples/simple-cluster.yaml#L19). To be on safe side, reverted it back.